sssom/cli.py

```diff
@@ -315,15 +312,15 @@ def dosql(query: str, inputs: List[str], output: str):
 )
 @click.option("-l", "--limit", type=int)
 @click.option("-P", "--prefix", type=click.Tuple([str, str]), multiple=True)
-@output_option
+@improved_output_option
 def sparql(
-    url: str = None,
-    config=None,
-    graph: str = None,
-    limit: int = None,
-    object_labels: bool = None,
```

Comment:
Emphatically do not add default values in `click` command line functions - click will always provide all of them

Reply:
click options get none by default anyway via the framework

Reply:
Excellent! mypy was complaining about this too! I've never initialized variables to `None` like this ever but again, this is code I inherited so no idea where this style came from.